### ISSUE TITLE
traing inception issue, validation low

### ISSUE THREAD

**Issue by GeorgeXia1828** [dmlc/mxnet#4745]
as once the issue https://github.com/dmlc/mxnet/issues/3032
and https://github.com/dmlc/mxnet/issues/3378
I am afraid the problem is still exist!

I am trying to retry the classification using network like inception bn full as I have done in mxnet 5.1 successful, but I have met some problem in mxnet latest, also I have raise a issue before. 
I have tried with fix gamma = true and false both failed .
**I found, 1. the train acc is not stable as training in mxnet 5.1 (which increase very small but very stable increase), but in the latest increase and decrease very unstable, and after one epoch increase to 16% around then quick drop back to very low!** 
for example:
 1 INFO:root:start with arguments Namespace(batch_size=128, benchmark=0, data_nthreads=4, data_train='/data5/rd/xiajizhong/parts_17kClasses/small_rec_test/', data_val='/data5/rd/x     iajizhong/parts_17kClasses/small_rec_val/', disp_batches=20, gpus='2,3', image_shape='3,224,224', kv_store='device', load_epoch=None, lr=0.5, lr_factor=0.9, lr_step_epochs='2,3     ,4,5,6,7,8,9,10,11,12,13,14,15', max_random_aspect_ratio=0.25, max_random_h=36, max_random_l=50, max_random_rotate_angle=10, max_random_s=50, max_random_scale=1, max_random_she     ar_ratio=0.1, min_random_scale=1, model_prefix='models/inception-bn-full', mom=0.9, monitor=0, network='inception-bn-full', num_classes=5190, num_epochs=80, num_examples=140400     , num_layers=50, optimizer='sgd', pad_size=0, random_crop=1, random_mirror=1, rgb_mean='123.68,116.779,103.939', test_io=0, top_k=0, wd=0.0001)
   2 [15:40:29] src/io/iter_image_recordio.cc:221: ImageRecordIOParser: /data5/rd/xiajizhong/parts_17kClasses/small_rec_test/, use 4 threads for decoding..
   3 [15:40:31] src/io/iter_image_recordio.cc:221: ImageRecordIOParser: /data5/rd/xiajizhong/parts_17kClasses/small_rec_val/, use 4 threads for decoding..
   4 INFO:root:Epoch[0] Batch [20]   Speed: 44.90 samples/sec    Train-accuracy=0.023810
   5 INFO:root:Epoch[0] Batch [40]   Speed: 53.99 samples/sec    Train-accuracy=0.042578
   6 INFO:root:Epoch[0] Batch [60]   Speed: 52.16 samples/sec    Train-accuracy=0.062500
   7 INFO:root:Epoch[0] Batch [80]   Speed: 49.40 samples/sec    Train-accuracy=0.052734
   8 INFO:root:Epoch[0] Batch [100]  Speed: 48.92 samples/sec    Train-accuracy=0.072266
   9 INFO:root:Epoch[0] Batch [120]  Speed: 49.46 samples/sec    Train-accuracy=0.069531
  10 INFO:root:Epoch[0] Batch [140]  Speed: 49.66 samples/sec    Train-accuracy=0.071875
  11 INFO:root:Epoch[0] Batch [160]  Speed: 49.06 samples/sec    Train-accuracy=0.073047
  12 INFO:root:Epoch[0] Batch [180]  Speed: 48.88 samples/sec    Train-accuracy=0.063672
....
 725 INFO:root:Epoch[1] Batch [3180] Speed: 54.00 samples/sec    Train-accuracy=0.163281
 726 INFO:root:Epoch[1] Batch [3200] Speed: 54.12 samples/sec    Train-accuracy=0.169531
 727 INFO:root:Epoch[1] Batch [3220] Speed: 53.97 samples/sec    Train-accuracy=0.176563
 728 INFO:root:Epoch[1] Batch [3240] Speed: 53.96 samples/sec    Train-accuracy=0.175391
 729 INFO:root:Epoch[1] Batch [3260] Speed: 54.00 samples/sec    Train-accuracy=0.103125
 730 INFO:root:Update[14249]: Change learning rate to 1.41215e-01
 731 INFO:root:Epoch[1] Batch [3280] Speed: 53.90 samples/sec    Train-accuracy=0.071094
 732 INFO:root:Epoch[1] Batch [3300] Speed: 53.86 samples/sec    Train-accuracy=0.068359
 733 INFO:root:Epoch[1] Batch [3320] Speed: 53.89 samples/sec    Train-accuracy=0.069531
 734 INFO:root:Epoch[1] Batch [3340] Speed: 54.10 samples/sec    Train-accuracy=0.064844
 735 INFO:root:Epoch[1] Batch [3360] Speed: 54.07 samples/sec    Train-accuracy=0.064844
 736 INFO:root:Epoch[1] Batch [3380] Speed: 54.16 samples/sec    Train-accuracy=0.065625
 737 INFO:root:Epoch[1] Batch [3400] Speed: 54.15 samples/sec    Train-accuracy=0.061328
 738 INFO:root:Epoch[1] Batch [3420] Speed: 54.18 samples/sec    Train-accuracy=0.071094
 739 INFO:root:Epoch[1] Batch [3440] Speed: 54.13 samples/sec    Train-accuracy=0.060547
 740 INFO:root:Epoch[1] Batch [3460] Speed: 54.11 samples/sec    Train-accuracy=0.064453
 741 INFO:root:Epoch[1] Batch [3480] Speed: 54.20 samples/sec    Train-accuracy=0.076172
 742 INFO:root:Epoch[1] Batch [3500] Speed: 54.03 samples/sec    Train-accuracy=0.064062
 743 INFO:root:Epoch[1] Batch [3520] Speed: 54.23 samples/sec    Train-accuracy=0.070312
 744 INFO:root:Epoch[1] Batch [3540] Speed: 54.11 samples/sec    Train-accuracy=0.056250
 745 INFO:root:Epoch[1] Batch [3560] Speed: 54.19 samples/sec    Train-accuracy=0.076953
 746 INFO:root:Epoch[1] Batch [3580] Speed: 54.13 samples/sec    Train-accuracy=0.067187
 747 INFO:root:Epoch[1] Batch [3600] Speed: 54.18 samples/sec    Train-accuracy=0.064062
 748 INFO:root:Epoch[1] Batch [3620] Speed: 54.20 samples/sec    Train-accuracy=0.075000
 749 INFO:root:Epoch[1] Batch [3640] Speed: 54.12 samples/sec    Train-accuracy=0.066797
